### PR TITLE
Updated readme with demo examples

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Evaluate observability platforms, like [Robusta.dev](https://home.robusta.dev/),
 
 # Usage
 Run various `kubectl apply` commands. To cleanup, run `kubectl delete` on the same.
-
+[label](blob:https%3A//twitter.com/90ebc1bd-acd5-4413-99ff-211efd1be12e)
```
Comment (aantn) [inline README.md:8]: ?

File: example_images/crashingpod.png
Comment (aantn) [inline example_images/crashingpod.png:1]: @pavangudiwada can we redo this one so there isn't empty whitespace at the top?

**Review by aantn — APPROVED** [review]
Looks great. Left two comments. Please fix and merge.

File: README.md (hunk 1 above)
Comment (pavangudiwada) [reply README.md:8]: Ahm....removed.

File: example_images/crashingpod.png
Comment (pavangudiwada) [reply example_images/crashingpod.png:1]: I can try.

Comment (pavangudiwada) [reply example_images/crashingpod.png:1]: Not perfect, but I believe it works.

**Pull request merged**